### DISCUSSION TITLE
GTC'22 Spring Cutlass Talks

### DISCUSSION THREAD

**Discussion by hwu36** [NVIDIA/cutlass#427] [General]
There are four Cutlass talks in the coming GTC

- Accelerating Backward Data Gradient by Increasing Tensor Core Utilization in CUTLASS [[S41996](https://events.rainfocus.com/widget/nvidia/gtcspring2022/sessioncatalog/session/1638923578147001lbbd)]: @manishucsd will introduce our super fast strided dgrad which can also be used as dconv as shown in #413.  He will also give a general CUTLASS update.  If you use CUTLASS heavily, don't miss it.
- Connect with the Experts: NVIDIA Math Libraries [[CWE41721](https://nam11.safelinks.protection.outlook.com/?url=https%3A%2F%2Freg.rainfocus.com%2Fflow%2Fnvidia%2Fgtcspring2022%2Faplive%2Fpage%2Fap%2Fsession%2F1638547181838001UPJ5&data=04%7C01%7Chaichengw%40nvidia.com%7C12420b48a2ec48033cb508da00588dc5%7C43083d15727340c1b7db39efd9ccc17a%7C0%7C0%7C637822677721044233%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=iuGJbGpSPgIrZ%2FtrY16XwcPOwhzieBHNZikxrRCPedU%3D&reserved=0)], @kerrmudgeon , @manishucsd , and @hwu36 will do live Q&A about CUTLASS.  Last year, we answered many questions from many users in a similar event.
- Use CUTLASS to Fuse Multiple GEMMs to Extreme Performance [[S41606](https://nam11.safelinks.protection.outlook.com/?url=https%3A%2F%2Freg.rainfocus.com%2Fflow%2Fnvidia%2Fgtcspring2022%2Faplive%2Fpage%2Fap%2Fsession%2F163843025136100131zi&data=04%7C01%7Chaichengw%40nvidia.com%7C12420b48a2ec48033cb508da00588dc5%7C43083d15727340c1b7db39efd9ccc17a%7C0%7C0%7C637822677721044233%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=ZxALtqW6SpfTsUXud2N6X1VqpXqeBz60Fmrl5BlZJW4%3D&reserved=0)]
- Auto48: A General Framework for Automatic Model Compression and Acceleration using Int4/Int8 Mixed Precision [[S41611](https://nam11.safelinks.protection.outlook.com/?url=https%3A%2F%2Freg.rainfocus.com%2Fflow%2Fnvidia%2Fgtcspring2022%2Faplive%2Fpage%2Fap%2Fsession%2F1638440164914001XUNd&data=04%7C01%7Chaichengw%40nvidia.com%7C12420b48a2ec48033cb508da00588dc5%7C43083d15727340c1b7db39efd9ccc17a%7C0%7C0%7C637822677721044233%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=jT%2FdGSUnbFKWsvT3kTgscf9lHgZ7%2BJxtNwpvRD9MV40%3D&reserved=0)]
- Large Models Are Not Always Expensive: Large Scale Mixture Of Expert Models With Efficient Inference Empowers MicroSoft Translator With Best Models [[S42518](https://reg.rainfocus.com/flow/nvidia/gtcspring2022/aplive/page/ap/session/1642794034438001oO4J)]

**Comment by masahi** [comment]
@hwu36 The link in `Accelerating Backward Data Gradient by Increasing Tensor Core Utilization in CUTLASS` above is pointing to a different talk.

Just watched three presentations related to the Hopper arch. Can't wait to see how cutlass would utilize all the cool new features!!
